auto_dev/commands/augment.py

```diff
@@ -466,8 +492,37 @@ def skill_from_fsm(spec_file: str, skill_public_id: PublicId, auto_confirm: bool
 
     output = scaffolder.scaffold()
     logger.info(f"Skill scaffolded: {output}")
+    write_to_file(behaviour_path, output, FileType.PYTHON)
+    logger.info(f"Behaviours file updated: {behaviour_path}")
+
+    # Extract the FSM behavior class name
+    class_name = None
+    for line in output.splitlines():
+        if line.startswith("class ") and "FsmBehaviour" in line:
+            class_name = line.split("(")[0].replace("class ", "").strip()
+            break
+
+    if not class_name:
+        logger.error("Could not find FSM behavior class name in generated code")
+        sys.exit(1)
 
-    # We now need to update the behvaiours.py file with the new handler
+    skill_yaml = read_from_file(skill_dir / "skill.yaml", FileType.YAML)
+    skill_yaml["behaviours"] = {
+        "main": {
+            "args": {},
+            "class_name": class_name,
+        }
+    }
+    write_to_file(skill_dir / "skill.yaml", skill_yaml, FileType.YAML)
+    logger.info(f"Skill.yaml updated: {skill_dir / 'skill.yaml'}")
+    ctx = Context(
+        cwd=Path.cwd(),
+        verbosity="info",
+        registry_path=Path.cwd() / "vendor",
+    )
+    fingerprint_item(ctx, str(PackageType.SKILL), skill_public_id)
+    package_manager = PackageManager(verbose=verbose)
```

Comment:
use a workflow for the dmeo :)

Reply:
@8ball030 do this here https://github.com/8ball030/auto_dev/pull/541